platform/darwin/src/MGLStyle.mm

```diff
@@ -92,6 +93,10 @@ - (NSString *)name {
     return @(self.mapView.mbglMap->getStyleName().c_str());
 }
 
+- (NSURL *)URL {
```

Comment:
What I was thinking of in https://github.com/mapbox/mapbox-gl-native/pull/6636#pullrequestreview-3370778 is just a comment here, something like: 

```objc
    // note that if `mbglMap->setStyleJSON()` is ever wired up, this can return unexpected results
```